iphone/Classes/KrollBridge.m

```diff
@@ -555,6 +555,16 @@ - (void)didStartNewContext:(KrollContext *)kroll
   // Make the global object itself available under the name "global"
   TiStringRef globalPropertyName = TiStringCreateWithCFString((CFStringRef) @"global");
   TiObjectSetProperty(jsContext, globalRef, globalPropertyName, globalRef, kTiPropertyAttributeDontEnum | kTiPropertyAttributeReadOnly | kTiPropertyAttributeDontDelete, NULL);
+  TiStringRelease(globalPropertyName);
+
+  // Set the __dirname and __filename for the app.js.
+  // For other files, it will be injected via the `TitaniumModuleRequireFormat` property
+  TiStringRef dirnameProperty = TiStringCreateWithCFString((CFStringRef) @"__dirname");
+  TiStringRef filenameProperty = TiStringCreateWithCFString((CFStringRef) @"__filename");
+  TiObjectSetProperty(jsContext, globalRef, dirnameProperty, [KrollObject toValue:kroll value:@"/"], kTiPropertyAttributeDontEnum | kTiPropertyAttributeReadOnly | kTiPropertyAttributeDontDelete, NULL);
+  TiObjectSetProperty(jsContext, globalRef, filenameProperty, [KrollObject toValue:kroll value:@"/app.js"], kTiPropertyAttributeDontEnum | kTiPropertyAttributeReadOnly | kTiPropertyAttributeDontDelete, NULL);
```

Comment:
`__filename` should just be the filename, no `/`.

Reply:
It's with parity of the current implementation (for require'ish format). Then we should change it for both.